File: ACTIVIDADESDELADSO2928797/Trabajo_en_clase/dart/programacion_orientada_a_objeto/31-07-2024/ejercicio2.dart
class Aprendiz{
  int? documento;
  int? seguro;
  int? ficha;

  Aprendiz(int d, int s){
    this.documento = d;
    this.seguro = s;
  }
}

void main(){
  Aprendiz laura = new Aprendiz(1042249249, 76543);
  laura.ficha = 2928707;
  print(laura.documento);
  print(laura.seguro);
  print(laura.ficha);

}

File: ACTIVIDADESDELADSO2928797/Trabajo_en_clase/Ruby/progorientadaobjeto/31julio2024/ejercicio2.rb
class Aprendiz
    attr_accessor :documento, :seguro, :ficha
    def initialize(d, s)
            self.d = documento
            self.s = seguro
    end
end

laura = Aprendiz.new(1042249249,3456)
laura.ficha = 2928707
puts laura.documento
puts laura.seguro
puts laura.ficha
